learning-team/2020-08/2020-08-06.md

```diff
@@ -0,0 +1,58 @@
+# Month Day, Year
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+### Concretize making PRs for 2 Ember Guides issues (IL, 15-20 min)
+
+IL - I think these feature requests will help in welcoming new visitors to the site. They do involve changing multiple files, possibly across multiple versions, so I'd like us to discuss how to divide the work involved so that each PR can be of reasonable size for review.
+
+- [Replace "First" and "Last" name code examples](https://github.com/ember-learn/guides-source/issues/1480)
+- [Fix API links in the Guides that go to "release"](https://github.com/ember-learn/guides-source/issues/1440)
```

Comment:
```suggestion
### Concretize making PRs for 2 Ember Guides issues (IL, 15-20 min)

IL - I think these feature requests will help in welcoming new visitors to the site. They do involve changing multiple files, possibly across multiple versions, so I'd like us to discuss how to divide the work involved so that each PR can be of reasonable size for review.

- [Replace "First" and "Last" name code examples](https://github.com/ember-learn/guides-source/issues/1480)
- [Fix API links in the Guides that go to "release"](https://github.com/ember-learn/guides-source/issues/1440)
```
<!-- replace with topic -->